packages/cardano-services/test/NetworkInfo/NetworkInfoHttpService.test.ts

```diff
@@ -339,7 +339,7 @@ describe('NetworkInfoHttpService', () => {
 
       it('successful request', async () => {
         const response = await provider.currentWalletProtocolParameters();
-        expect(response.maxTxSize).toBeDefined();
+        expect(response?.maxTxSize).toBeDefined();
```

Comment:
This looks suspicious, is this needed? :thinking:

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/388/commits/723a72ecc4fe7bc976472a733eed0c47e0fbaf38